ember-qunit-codemod.js

```diff
@@ -222,6 +222,23 @@ function updateModuleForToNestedModule(j, root) {
   bodyPath.replace(bodyReplacement);
 }
 
+function updateLookupCalls(j, root) {
+  return root
+    .find(j.MemberExpression)
```

Comment:
I think you can modify this to remove the manual expression filtering (in favor of matching via the `.find` call):

```js
root.find(j.MemberExpression, {
  object: {
    object: {
      type: 'ThisExpression',
    },
    property: {
      name: 'container'
    }
  },
  property: {
    name: 'lookup'
  }
})
.forEach(path => {
  let thisDotOwner = j.memberExpression(j.thisExpression(), j.identifier('owner'));
  path.replace(j.memberExpression(thisDotOwner, path.value.property));
})
```

Reply:
I never tried that syntax, give some minutes